clang/lib/CIR/CodeGen/CIRGenFunction.cpp

```diff
@@ -370,22 +370,62 @@ void CIRGenFunction::LexicalScope::cleanup() {
     (void)buildReturn(retLoc);
   }
 
+  auto buildImplicitReturn = [&]() {
```

Comment:
This function is already a bit big, instead of the lambda, please break this apart into a new one!

Reply:
Updated.